.gitignore

```diff
@@ -3,3 +3,6 @@
 .RData
 revdep/checks
 revdep/library
+*.o
+*.so
+issues
```

Comment:
Shouldn't `.o` and `.so` be in your global gitignore? No worries though.

What's the `issues` file?

Reply:
I sometimes put .R files in there when I'm messing with code to reproduce issues. I can move both things to a global gitignore for sure.

Reply:
makes sense. started using `issues/` because it was already git-ignored by @krlmlr in the dplyr repo. 

Do you want me to revert this change here ?

Reply:
No worries you can leave it. In general it may be better to use an approach that doesn't require changing the setup of repos.

Reply:
I just noticed `usethis::use_rcpp()` adds the binary files to gitignore so you were right to add them:

```
✔ Adding '*.o', '*.so', '*.dll' to 'src/.gitignore'
```